src/utils/filteredTodos.tsx

```diff
@@ -0,0 +1,22 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export const FilteredTodos = (todos: Todo[], filterBy: Status): Todo[] => {
+  let filteredTodos = [...todos];
```

Comment:
In this case, you don't need let . You can use const and then return operations with the array.